test/integration/clusterquota_test.go

```diff
@@ -189,5 +190,10 @@ func waitForQuotaStatus(clusterAdminClient quotaclient.Interface, name string, c
 		}
 		return false, nil
 	})
-
+	if err == nil {
+		// since now we run each process separately we need to wait for the informers
+		// to catch up on the update and only then continue
+		time.Sleep(3 * time.Second)
```

Comment:
omfg.  The fact that I'll take this is a sign of how desperate I am.  Make an issue on our jira board to remove this somehow.

Reply:
https://jira.coreos.com/browse/MSTR-222